en-US/focus-ios.xliff

```diff
@@ -931,6 +930,26 @@
         <target>Site missing content or acting strange?</target>
         <note>Text for a label that indicates the title for sites not working tip.</note>
       </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionGalleryDescription">
+        <source>Add a Firefox shortcut to your Home screen. After adding the widget, touch and hold to edit it and select a different shortcut.</source>
```

Comment:
As mentioned before, this shouldn't mention `Firefox`. It should be a placeable with either Focus or Klar determined at runtime.

Reply:
Should be fixed by #3435